libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKAuthRootController.m

```diff
@@ -26,17 +26,9 @@
 */
 
 #import "SFSDKAuthRootController.h"
-#import <AuthenticationServices/AuthenticationServices.h>
 
 @implementation SFSDKAuthRootController
-
-/**
-  Implementating  ASWebAuthenticationPresentationContextProviding so that ASWebAuthenticationSession can be started from the window
- */
-#if __IPHONE_OS_VERSION_MIN_REQUIRED >= 130000
```

Comment:
@trooper2013 , we are using deployment target to check? not the actual OS version? Is that actually the proper way to to do this?